xarray/tests/test_array_api.py

```diff
@@ -43,6 +43,12 @@ def test_indexing(arrays) -> None:
     assert_equal(actual, expected)
 
 
+def test_properties(arrays) -> None:
+    np_arr, xp_arr = arrays
+    assert np_arr.nbytes == 48
+    assert xp_arr.nbytes == 48
```

Comment:
```suggestion
    assert np_arr.nbytes == np_arr.data.nbytes
    assert xp_arr.nbytes == np_arr.data.nbytes
```
I think at some point xp might not always be the array api and then the value could change.